android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,9 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<network-security-config>
-    <base-config cleartextTrafficPermitted="true">
+<network-security-config xmlns:tools="http://schemas.android.com/tools">
+    <base-config cleartextTrafficPermitted="false">
         <trust-anchors>
             <certificates src="system" />
-            <certificates src="user" />
+            <certificates src="user"
+                tools:ignore="AcceptsUserCertificates" />
         </trust-anchors>
     </base-config>
+    <debug-overrides>
```

Comment:
It looks like it's not working.
You can try running your app on your device with: `npx react-native run-android --variant=playDebug`
You're going to get this screen:
<img src="https://user-images.githubusercontent.com/804994/91495017-3a57f900-e890-11ea-8d69-171ab1fc6468.png" width="350" />

That's because the JS bundle is sent to the device via port 8081 and it needs `cleartextTrafficPermitted="true"`.
Apparently, `<debug-overrides>` isn't really doing anything.
If I got it right, `<debug-overrides>` can't contain a `<base-config>`: https://developer.android.com/training/articles/security-config#debug-overrides

Reply:
I'll take a look - I forgot about how RN debugging works as I tried with a production build. 👍